src/sagemaker/serve/builder/jumpstart_builder.py

```diff
@@ -234,6 +254,213 @@ def _build_for_tgi_jumpstart(self):
 
         self.pysdk_model.env.update(env)
 
+    def _tune_for_js(
+        self, multiple_model_copies_enabled: bool = False, max_tuning_duration: int = 1800
+    ):
+        """Tune for Jumpstart Models.
+
+        Args:
+            multiple_model_copies_enabled (bool): Whether multiple model copies serving is enable by
+                this ``DLC``. Defaults to ``False``
+            max_tuning_duration (int): The maximum timeout to deploy this ``Model`` locally.
+                Default: ``1800``
+        returns:
+            Tuned Model.
+        """
+        if self.mode != Mode.LOCAL_CONTAINER:
+            logger.warning(
+                "Tuning is only a %s capability. Returning original model.", Mode.LOCAL_CONTAINER
+            )
+            return self.pysdk_model
+
+        num_shard_env_var_name = "SM_NUM_GPUS"
+        if "OPTION_TENSOR_PARALLEL_DEGREE" in self.pysdk_model.env.keys():
+            num_shard_env_var_name = "OPTION_TENSOR_PARALLEL_DEGREE"
+
+        initial_env_vars = copy.deepcopy(self.pysdk_model.env)
+        admissible_tensor_parallel_degrees = _get_admissible_tensor_parallel_degrees(
+            self.js_model_config
+        )
+        available_gpus = _get_available_gpus() if multiple_model_copies_enabled else None
```

Comment:
Minor just remove this line and combine it with setting the sagemaker_model_server_workers since you are already checking if mutiple_model_copies_enabled is True there